__tests__/unit/core-forger/delegate.test.ts

```diff
@@ -13,9 +12,11 @@ const dummy = {
 };
 
 describe("Delegate", () => {
+    beforeAll(() => Managers.configManager.setFromPreset("testnet"));
```

Comment:
I think this should be "unitnet", not "testnet". That file is in  tests/`unit`/core-forger/delegate.test.ts after all.

Using testnet in unit tests instead of unitnet creates problems later with milestones - if a test should be written that uses some big height.

But I see that this test is already using "testnet", so maybe fixing it to use "unitnet", if too involved, would be out of the scope of this PR.